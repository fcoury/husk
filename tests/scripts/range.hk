let arr = [1, 2, 3, 4, 5];

println!(arr[2..3]);
println!(arr[2..=3]);
println!(arr[3..]);
println!(arr[..2]);
println!(arr[..=2]);

