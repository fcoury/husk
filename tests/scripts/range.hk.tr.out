[ 3 ]
[ 3, 4 ]
[ 4, 5 ]
[ 1, 2 ]
[ 1, 2, 3 ]
